apps/climatemappedafrica/src/components/RichHeader/RichHeader.js

```diff
@@ -50,8 +50,8 @@ const RichHeader = React.forwardRef(function RichHeader(props, ref) {
               position: "absolute",
               width: "100%",
             },
-            ...TitleProps?.sx,
           },
+          ...TitleProps?.sx,
```

Comment:
Again, why?

Reply:
@kilemensi maybe I missed something here but should sx prop be spread under `"& strong"` or on `sx`.
I thought we intended to have this 👇👇
![image](https://github.com/user-attachments/assets/a31da6c6-aa1c-412f-a093-ef058da4061f)

Reply:
Should 100% be under `sx` @koechkevin, may be it's me who is seeing doubles!